ui/app/webpack.common.ts

```diff
@@ -35,6 +35,7 @@ export const commonConfig: Configuration = {
     new ForkTsCheckerWebpackPlugin({
       typescript: {
         configFile: path.resolve(__dirname, './tsconfig.json'),
+        build: true, // Since we use project references...
```

Comment:
This _should_ fix the error that we see sometimes locally where TypeScript complains that "XXX has not been built from source file YYY".